shell/detail/networking.k8s.io.ingress.vue

```diff
@@ -18,76 +16,38 @@ export default {
   },
   mixins: [CreateEditView],
   async fetch() {
+    this.ingressHelper = new IngressDetailEditHelper({
+      $store:    this.$store,
+      namespace: this.value.metadata.namespace
+    });
     const promises = {
-      services:       this.$store.dispatch('cluster/findAll', { type: SERVICE }),
+      services:       this.ingressHelper.fetchServices(),
+      secrets:        this.ingressHelper.fetchSecrets(),

```

Comment:
I kind of wish we used this sort of pattern over using stores for all resource fetching.